tests/e2e-openshift/monitoring/check_user_workload_monitoring.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Why do we need this bash script? Is the number of user workload monitoring component replicas different in crc and prow setups and therefore we cannot assert a fixed number in a regular kuttl step?

Reply:
Yes that is correct, the monitoring CRC plugin only has 1 replica for the components while the OCP clusters have a minimum of 2 replicas. That is the reason we need the script.